scripts/tkg_byoi.py

```diff
@@ -101,6 +102,11 @@ def populate_jinja_args(args):
     jinja_args_map["kubernetes_version"] = jinja_args_map["kubernetes"]
     jinja_args_map["kubernetes_series"] = jinja_args_map["kubernetes"].split('+')[0]
 
+    # Set STIG compliant value
+    jinja_args_map["photon3_stig_compliance"] = "false"
```

Comment:
You can directly use Boolean here and also in Jinja (`True`, `False` and in Jinja `{% if photon3_stig_compliance %}`), or passing like that not working the Ansible 🤔 ?

Reply:
This is not working for ansible, hence I took the values as strings rather than boolean